WebApplication1/Controllers/UsersController.cs

```diff
@@ -24,32 +24,32 @@ public UsersController(ILogger<UsersController> logger)
             _logger = logger;
         }
 
-        [HttpPost]
-        public IActionResult AddUser(User user)
+        [HttpPost(Name = nameof(AddNewUser))]
```

Comment:
There is standard of request naming. In your case each method name need to be known by caller. It could be a problem sometimes.
I'd prefer to use common style as we did on lesson.

Reply:
done